debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresOffsetContext.java

```diff
@@ -55,7 +55,8 @@ private PostgresOffsetContext(PostgresConnectorConfig connectorConfig, Lsn lsn,
 
         this.lastCompletelyProcessedLsn = lastCompletelyProcessedLsn;
         this.lastCommitLsn = lastCommitLsn;
-        sourceInfo.update(lsn, time, txId, null, sourceInfo.xmin(), lastCommitLsn);
+        sourceInfo.update(lastCommitLsn, time, txId, null, sourceInfo.xmin(), true);
```

Comment:
See bellow, this sequence is a little bit confusing and having two differently named methods would clarify it a lot.